### PR TITLE
add ci for openshift/cluster-policy-controller 

### PR DESCRIPTION
**Pull request by sallyom**
* copied config file from openshift-controller-manager master branch ~& 4.3 branch~ to cluster-policy-controller (there is no 4.3 branch in cluster-policy-controller), updated accordingly

* added jobs for openshift/cluster-policy-controller

@deads2k @sanchezl

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ presubmits:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```
Comment (sallyom) [inline ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: true or false? must've been manually updated, as this was generated.

File: ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml [hunk 2]
```diff
@@ -76,9 +76,8 @@ presubmits:
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-openshift-controller-manager-master-e2e-aws-builds
+    name: pull-ci-openshift-cluster-policy-controller-master-e2e-aws-builds
     rerun_command: /test e2e-aws-builds
-    run_if_changed: ^(pkg/build)
```
Comment (sallyom) [inline ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: and probably want this, too

File: ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: Yeah you likely want to copy the `jobs/` files first and edit the repo name there, then regenerate

Comment (sallyom) [reply ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: fixed, got it

**Comment by tnozicka** [comment]
@sallyom I think the needs to be copy instead of rename

**Comment by sallyom** [comment]
> > renamed config file for master branch & 4.3 branch:
> > openshift-controller-manager -> cluster-policy-controller
> 
> @sallyom I think the needs to be copy instead of rename

ah I thought going forward this new repo would replace not co-exist, bc the Dockerfile info in cluster-policy-controller still indicates `openshift-controller-manager` : ) I'll fix that up now, restore the files.

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+- as: e2e-aws-upgrade
+  commands: TEST_SUITE=all run-upgrade-tests
+  openshift_installer:
+    cluster_profile: aws
+    upgrade: true
+- as: e2e-aws-builds
```
Comment (tnozicka) [inline ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: I don't think we need builds here

Comment (sallyom) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: I've removed, also bumped the config to 4.3

Comment (sallyom) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: I've removed, and bumped to 4.3

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
+  to: cluster-policy-controller
+# TODO: wait for image to exist
```
Comment (tnozicka) [inline ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: promotion is fine even without ART (not related to CI) you just can't use the image in the code

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+#  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 9Gi
+    requests:
+      cpu: "3"
+      memory: 6Gi
+tag_specification:
+  name: "4.2"
```
Comment (deads2k) [inline ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: this doesn't have a 4.2

File: ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml (hunk 1 above)
Comment (tnozicka) [reply ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: I'd go for copy only config files and regenerate jobs, there shouldn't be manual edits

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml (hunk 5 above)
Comment (tnozicka) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: yes, should be using 4.3 images

File: ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml (hunk 1 above)
Comment (sallyom) [reply ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml]: i don't have any manual edits, it's good

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml (hunk 4 above)
Comment (sallyom) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: oh ok, i've uncommented

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml (hunk 5 above)
Comment (sallyom) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: oops missed that, fixed

Comment (sallyom) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: missed that, fixed

**Pull request merged**